homeassistant/components/rss_feed_template.py

```diff
@@ -0,0 +1,93 @@
+"""
+Exports sensor values via RSS feed.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/rss_feed_template/
+"""
+
+import asyncio
+from html import escape
+from aiohttp import web
+
+import voluptuous as vol
+
+from homeassistant.components.http import HomeAssistantView
+import homeassistant.helpers.config_validation as cv
+
+DOMAIN = "rss_feed_template"
+DEPENDENCIES = ['http']
+
+CONTENT_TYPE_XML = "text/xml"
+
+CONFIG_SCHEMA = vol.Schema({
+    DOMAIN: cv.ordered_dict(
+        vol.Schema({
+            vol.Optional('title'): cv.template,
+            vol.Required('items'): vol.All(
+                cv.ensure_list,
+                [{
+                    vol.Optional('title'): cv.template,
+                    vol.Optional('description'): cv.template,
+                }]
+                )
+            })
+        )
+    }, extra=vol.ALLOW_EXTRA)
+
+
+def setup(hass, config):
+    """Setup the RSS feeds."""
+    for (feeduri, feedconfig) in config[DOMAIN].items():
+        rss_view = RssView(hass, feedconfig)
+        hass.http.app.router.add_route('get',
+                                       '/api/rss/%s' %
+                                       feeduri, rss_view.get)
+
+    return True
+
+
+class RssView(HomeAssistantView):
+    """Export states and other values as RSS."""
+
+    _title = None
+    _items = None
+
+    def __init__(self, hass, feedconfig):
+        """Initialize the rss view."""
+        self._title = feedconfig.get('title')
+        if self._title is not None:
+            self._title.hass = hass
+
+        self._items = feedconfig.get('items')
+        for item in self._items:
+            if 'title' in item:
+                item['title'].hass = hass
```

Comment:
Can you set `hass` on the templates inside the setup methods? That way you don't need to pass in hass either.